### PR TITLE
Transfer WikiCanada to Elements wiki

### PR DESCRIPTION
**Pull request by DeltaQuad-DQ**
Hope I did this correctly

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2719,6 +2718,21 @@
...
+				'moderator',
+				'sysop',
+				'bot',
+				'confirmed',
+				'autopatrolled',
+				'rollbacker',
+				'tagmanager',
+			),
+		),
+
```
Comment (Reception123) [inline LocalSettings.php]: please remove this unnecessary space

File: LocalSettings.php [hunk 2]
```diff
@@ -3364,6 +3365,21 @@
...
+				'founder',
+				'moderator',
+				'sysop',
+				'bot',
+				'confirmed',
+				'autopatrolled',
+				'rollbacker',
+				'tagmanager',
+			),
+
```
Comment (Reception123) [inline LocalSettings.php]: remove this space

File: LocalSettings.php [hunk 3]
```diff
@@ -3445,21 +3461,6 @@
 		'+wikidolphinhansenwiki' => array(
 			'sysop' => array(
 				'commentadmin',
-			),
```
Comment (Reception123) [inline LocalSettings.php]: there is no reason to delete this array and the one below

File: LocalSettings.php [hunk 4]
```diff
@@ -3445,21 +3461,6 @@
...
-				'founder',
-				'moderator',
-				'sysop',
-				'bot',
-				'confirmed',
-				'autopatrolled',
-				'rollbacker',
-				'tagmanager',
-			),
 		),
```
Comment (Reception123) [inline LocalSettings.php]: this array also needs to be deleted as there are two arrays above

File: LocalSettings.php [hunk 5]
```diff
@@ -3469,20 +3470,20 @@
...
+			
+		),
 		'testwiki' => array(
 			'sysop' => array(
 				# 'nuke' => true, // done in overrides at end of file
 				# 'editinterface' => true, //mistakenly applies to other groups as well
 			),
 			'exampleuser' => array(
 				'editmyoptions' => true,
-			),
```
Comment (Reception123) [inline LocalSettings.php]: again please leave these arrays alone

File: LocalSettings.php [hunk 6]
```diff
@@ -3469,20 +3470,20 @@
...
-		),
-		'wikicanadawiki' => array(
- 			'banned' => array(
- 				'read' => true,
-				'edit' => true,
- 			),
 			
  		),
```
Comment (Reception123) [inline LocalSettings.php]: and delete this array too

**Comment by Reception123** [comment]
You could have simply replaced the names of the wikis and it would have been easier.

**Comment by DeltaQuad-DQ** [comment]
@Reception123 I thought the wikis need to be in alphabetical order

**Comment by Reception123** [comment]
Sorry, yes you are right. You still need to fix your errors.

**Comment by DeltaQuad-DQ** [comment]
Can you/someone else re-do the PR's for me? I'm not exactly sure how to fix everything

**Comment by DeltaQuad-DQ** [comment]
I don't know how to fix these errors myself. I will need @Reception123 or someone else to re-do this request for me.

**Pull request closed without merging**